### PR TITLE
[luci] Extract common ConvertNCHWToNHWCPass

### PR DESCRIPTION
**Pull request by seanshpark**
This will extract common code in ConvertNCHWToNHWCPass with Relu type.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
For #6053, from #6054,

File: compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp [hunk 1]
```diff
@@ -369,6 +369,26 @@ bool is_NCHW_with_const(const luci::CircleAdd *node, luci::CircleNode *&pred_nod
     return false;
 }
 
+template <class T> bool convert_Relu(T *node)
```
Comment (jinevening) [inline compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:372]: How about naming this `convert_single_input_operator` and including Neg too?

`node->features()` in the function body may be replaced with `node->args(0)`.

Comment (seanshpark) [reply compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:372]: how about shorter `convert_unary` ?

File: compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp [hunk 2]
```diff
@@ -369,6 +369,26 @@ bool is_NCHW_with_const(const luci::CircleAdd *node, luci::CircleNode *&pred_nod
...
+{
+  const auto pred_node = loco::must_cast<luci::CircleNode *>(node->features());
+  auto pre_trans = create_pre_transpose(node);
+  pre_trans->a(pred_node);
+  node->features(pre_trans);
```
Comment (seanshpark) [inline compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:377]: we cannot use like `node->args(0) = pre_trans;`

File: compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp:372]: for future, I'll update this method as `convert_unary_features`

Comment (jinevening) [reply compiler/luci/pass/src/ConvertNCHWToNHWCPass.cpp]: > We cannot use node->args(0) as setter. so cannot merge with Neg

Oh I see. Thank you.

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**